book/webspaces.rst

```diff
@@ -55,6 +55,21 @@ following paragraphs.
             </contexts>
         </navigation>
 
+        <segments>
```

Comment:
I would not add `<segments>` XML Tag here to the basic webspace example we use here in the install example. As I think its an edge case and we also do not added `<security>` and `<theme>` here in the basic book example.

Reply:
I think that this is mixed here. If we really want to keep it minimal here, we would probably also have to remove the `excluded-templates` and `resource-locator` tags. But they are also here and only marked as "optional" in their headings. And since there is currently no other place where we are explaining the Webspace XML file I don't really see where else we should put it.